Dockerfile

```diff
@@ -51,10 +51,10 @@ RUN  wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-ke
   && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
```

Comment:
I think if we switch to using the `chromium`, then the chrome apt source can be removed.